EpsilonDriverControls/Src/DriverControls.c

```diff
@@ -206,12 +206,6 @@ void sendDriverTask(void const* arg)
 void sendDriveCommandsTask(void const* arg)
 {
     uint32_t prevWakeTime = osKernelSysTick();
-    float newRegen = 0;
-    float newAccel = 0;
-    uint8_t forward = 0;
-    uint8_t reverse = 0;
-    uint8_t brake = 0;
-    uint8_t reset = 0;
     float busCurrentOut = 1.0f; // Percentage 0 -1 always 100%
```

Comment:
I think this value never changes at all, so it can maybe be changed to a macro instead?